README.md

```diff
@@ -1,5 +1,5 @@
 # REopt Julia package
-This package is currently under development, but it now has **most** of capabilities of the REopt model used in the [REopt API](https://github.com/NREL/REopt_API). We expect to have the first stable release by Fall 2022.
+This package is currently under development, but it now has all of the capabilities of the REopt model used in the [REopt API](https://github.com/NREL/REopt_API). We expect to have the first stable release by Summer 2023.
```

Comment:
@bpulluta are we sure it currently has all? I believe we're still missing a few techs, although they should be in there by the Summer 2023 release

Reply:
@adfarth See b37fcde8f592c3736e58e5c838b82fc021919. It seems that this was an update from 2-weeks ago. Should I revert to the previous description?

Reply:
@adfarth not sure how this ended up in my branch but I reverted back to the old timeline